### PR TITLE
Adds vox mining, mechanic, and roboticist space suits

### LINKED ISSUE #11173: Vox Miners do not spawn with a Miner hardsuit
#### Description of issue

Vox Miners do not spawn with a mining suit, instead have an assistant suit

Vox Mechanics have this issue as well
#### Difference between expected and actual behavior

Miner job on vox does not spawn with a job specific suit
#### Steps to reproduce

Spawn as Vox Miner

I've taken the time to nigger rig a set of clothes for players spawning as a Vox Miner in an attempt to solve this issue

[](url)

http://puu.sh/qh45B.dmi
http://puu.sh/qh46P.dmi

suit
head

### PR DESCRIPTION
**Pull request by zhanbow**
voxample2

Adds vox mining, mechanic, and roboticist space suits

Also allows miner suits to be goliath upgraded.

Fixes #11173

:cl:
 * rscadd: Added suits for vox miners, roboticists, and mechanics.

### CODE REVIEW

**Comment by SonixApache** [comment]
i want the furfag snowflakes to leave

**Comment by zhanbow** [comment]
ok

**Comment by kilozombie** [comment]
We already have vox pressure suits so why is it more snowflakey to just have ones for jobs that there aren't any currently
This is an objectively good addition because it means spess is more consistent and less bad

**Comment by SonixApache** [comment]
or we could just remove the pressure suits since they literally don't need it

**Comment by zhanbow** [comment]
this is not the pull request to argue about that

**Comment by kilozombie** [comment]
Vox can't wear any hardsuits beyond the ones they start with and also need a pressure suit to carry a backpack at all, but yeah, they clearly don't need it
Even if you really think Vox players are all furfags or snowflakes, why would you oppose pressure suits? They literally cover up all the features and make a faceless non-snowflake thing.

**Comment by Aurx** [comment]
>need a pressure suit to carry a backpack at all

Bull and shit, back when I played Vox I never wore my suit and always carried my backpack.  
It's trivial to just put the pack in your hand, or if you're too 2fast2hand2furious you can just carry a stack of emergency N2 tanks and keep an eye on your remaining gas.

**Comment by AutismSpider** [comment]
So re-code hard suits/soft suits so it fits the vox and touch up the icons so they don't stick out.

**Comment by kilozombie** [comment]
Sorry-- not carry, wear, you're right. If you're okay with the risk of losing all your stuff you can carry your backpack around.
There is nothing wrong with the pressure suits, they have high-quality sprites and make the species at least a little more alien. The only legitimate argument is whether or not species should have roundstart suits, but... plasmamen, homegrown.
And if you think suits are OP, just look at diona (no pressure damage at all because it's healed instantly)
I don't want to go 'but what about this?' too much, my point is that it's a petty thing to make trouble over, especially when the PR is legitimately a feature addition to a feature that already exists.

**Comment by Kurfursten** [comment]
More pressure suit variety is objectively a good thing.

You want to be able to immediately identify if the person standing inside the mechanic workshop is a mechanic or an assistant who broke in as an officer. This is why we have job specific uniforms in the first place.

It boggles my mind that people would argue against better uniform identifiability, it sounds to me like saying:

>Damn snowflake roboticists easily identifiable by their black jumpsuits and white labcoats. Why can't we all just wear grey?

Not to mention departmental suits make it much harder for Vox to pass off blame when committing a crime. Right now they can just take off their ID and blend in as an "assistant".

**Comment by kilozombie** [comment]
>Fucking snowflakes not playing as a bald man named "John Doe" wearing a grey jumpsuit

**Comment by yclatious** [comment]
Personally, I don't like integrating the birds anymore than they already are integrated, but kurfursten has a point, previous rants about excessive snowflake adaptability and contradiction of original design and crew interaction aside.

I don't like birds getting free things or snowflakes, I passionately hate the combination of both, but this is fine, even if potentially risking a slipery slope cascade of bullshit.

**Comment by zhanbow** [comment]
i just wanted to add some sprites

**Comment by kilozombie** [comment]
You have to deal with the bikeshed gauntlet every time you add anything to the game, this is a proven fact.

**Comment by yclatious** [comment]
You mean well, but sometimes people who mean well are stupid retards who don't think through about their changes and the consequences they make, like the pizza bomb TG retard who wanted to add a invisible pouch solely for snowflaking and though it was fine. Also, Im not calling you a stupid retard, just so Im being clear.

All changes have effects and can change things you may not have considered. I don't like it but I won't opose it. Also what kilo said, its ss13, confusion and conflict aren't just ingame things.

**Comment by Probe1** [comment]
If you want to remove vox pressure suits make an issue.  Stop starting fights in PRs.

File: code/modules/mob/living/carbon/species.dm [hunk 1]
```diff
@@ -676,6 +682,9 @@ var/global/list/whitelisted_species = list("Human")
 		if("Research Director")
 			suit=/obj/item/clothing/suit/space/vox/civ/science/rd
 			helm=/obj/item/clothing/head/helmet/space/vox/civ/science/rd
+		if("Roboticist")
+			suit=/obj/item/clothing/suit/space/vox/civ/science/roboticist
+			helm=/obj/item/clothing/head/helmet/space/vox/civ/science/roboticist
```
Comment (Probe1) [inline code/modules/mob/living/carbon/species.dm:687]: https://a.pomf.cat/lsoyiu.png

For some awful reason this isn't working.   Try pathing it as vox/civ/roboticist.

**Review by Probe1 — CHANGES_REQUESTED** [review]
Fix roboticist sprites defaulting to scientist sprites.

**Comment by Probe1** [comment]
Look at you go.  Next thing you know it you'll be testing your PRs before submitting them 😄

**Comment by zhanbow** [comment]
yeah, turns out it was part of an if statement further up, woops

it was the only one i didn't test as well

**Comment by Kurfursten** [comment]
>Testing PRs before submitting them

Wish we could turn back time
To the good old days
When we submit content
Without compiling

File: code/modules/mob/living/carbon/species.dm (hunk 1 above)
Comment (zhanbow) [reply code/modules/mob/living/carbon/species.dm:687]: fixed and tested them all in latest commit, should be fine now

**Review by Probe1 — APPROVED** [review]

**Comment by Probe1** [comment]
Travis is a crutch for lesser coders.  We are men amongst men.  Who needs a testing environment when you have production already?

**Pull request merged**